examples/pxScene2d/src/pxScene2d.h

```diff
@@ -1198,6 +1208,10 @@ class pxScene2d: public rtObject, public pxIView, public rtIServiceProvider
     return mArchive;
   }
 
+#ifdef PXSCENE_SUPPORT_STORAGE
+  rtError storage(rtObjectRef& v) const;
```

Comment:
Please ensure that all methods are declared without surrounding #ifdefs.  Please add the #ifdef checks inside the methods themselves.  If the define is not declared then please log a message but perform no action.  However, a method should be declared if the define is set or not.  This will prevent us from having to add new defines when building other software components that use Spark headers.  Please update for all other new methods as well.